.github/workflows/rust.yml

```diff
@@ -0,0 +1,36 @@
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types:
+
+jobs:
+  cargo_nextest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-22.04-8core
+            flags: ""
+          - os: ubuntu-22.04-8core
+            flags: "--features nightly,test_features"
```

Comment:
Do you happen to what's test_features?

Reply:
There are a number of `#[cfg]`s for this feature around the `chain/client`, `jsonrpc` code and such. I don't know what purpose it serves exactly, but it got carried over from buildkite so there must be a good point to testing this.